packages/cubejs-teradata-jdbc-driver/bin/installer.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.downloadJDBCDriver = void 0;
+const path_1 = __importDefault(require("path"));
+const inquirer_1 = __importDefault(require("inquirer"));
+const shared_1 = require("@cubejs-backend/shared");
+function acceptedByEnv() {
+    const acceptStatus = shared_1.getEnv('teradataAcceptPolicy');
+    if (acceptStatus) {
+        console.log('You accepted Terms & Conditions for JDBC driver from Teradata by CUBEJS_DB_TERADATA_ACCEPT_POLICY');
+    }
+    if (acceptStatus === false) {
```

Comment:
Not critical but I guess this is a copy-n-pasted code, I find it a bit odd to test for a boolean with == false, and use two if statements instead of if-else, but anyhow...

Reply:
I removed it because of previous PR comment